cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js

```diff
@@ -74,6 +77,37 @@ export default {
       });
   },
 
+  searchFieldMappingProfile(text) {
+    cy.do(searchField.fillIn(text));
+  },
+
+  verifySearchButtonEnabled: () => cy.expect(searchButton.has({ disabled: false })),
+  verifySearchButtonDisabled: () => cy.expect(searchButton.has({ disabled: true })),
```

Comment:
merge these into one method where a true or false argument is passed, like in the following example:
  isUsersRadioChecked(checked = true) {
    cy.expect(usersRadio.has({ checked }));
  },

Reply:
fixed